devel/libfilezilla/Portfile

```diff
@@ -40,12 +38,34 @@ depends_lib         port:libiconv \
                     port:nettle \
                     path:lib/pkgconfig/gnutls.pc:gnutls
 
+# Keep this until gnutls is updated to 3.7.0+
+depends_lib-delete  path:lib/pkgconfig/gnutls.pc:gnutls
+depends_lib-append  path:lib/pkgconfig/gnutls.pc:gnutls-devel
```

Comment:
is it required for every platform? If no, I suggest to move it to your if, otherwise it will be linked against gnutls-devel by default, that is bad idea.

Reply:
It is required by configure script of the source.

Reply:
@Schamschula looks like that update gnutls is required :(

Reply:
I started a thread on the gnutls mailing list about the lack of the security back port for the 3.6, i.e.  'Stable Branch'.

See: https://lists.gnutls.org/pipermail/gnutls-help/2022-August/004747.html

and https://lists.gnutls.org/pipermail/gnutls-help/2022-September/thread.html

I'm not going to declare the 3.7 branch stable unless upstream does so first.

Reply:
@barracuda156 seems you have the first blocker :)

Reply:
Sergey, is there a slightly older version of this port, that works with our current version of `gnutls`?

That's your only option, as we can't depend on `gnutls-devel`.

Reply:
> Sergey, is there a slightly older version of this port, that works with our current version of `gnutls`?
> 
> That's your only option, as we can't depend on `gnutls-devel`.

@mascguy I can check I guess, but why not though?

1. `libfilezilla` has _the only_ one dependent, `FileZilla`; so only those interested to have that specific port gonna bother to build it, and well, they could install `gnutls-devel` then.
2. I searched through Macports database, and we do have some ports which specifically depend on devel versions of some other port.

Reply:
Here's the problem: `gnutls` is a foundational port, and will likely be auto-installed [as a dependency of many other ports] the vast majority of the time.

So then, when the user tries to installed `FileZilla`, it will fail, because the users already have `gnutls` installed, and macports will not forcibly remove that in favor of `gnutls-devel`.

That's why it's very rare that we depend on Devel ports. And if you look more closely at the list of ports that do use them, they're typically also Devel ports.

Reply:
> @mascguy I can check I guess, but why not though?
> 
>     1. `libfilezilla` has _the only_ one dependent, `FileZilla`; so only those interested to have that specific port gonna bother to build it, and well, they could install `gnutls-devel` then.
> 
>     2. I searched through Macports database, and we do have some ports which specifically depend on devel versions of some other port.

So, you have two options:
* Find an older version that builds with `gnutls`
* Patch the code to work with `gnutls`

Reply:
@mascguy Got it, makes sense. I convert this to Draft first and let me think what to do with this.

Reply:
@mascguy I checked recent versions of libfilezilla, and 0.32.0 is the border case, it needs 3.6.14, while 0.33.0 needs 3.7.0. Will use 0.32.0 then.